src/app/products/manage-tax-groups/create-tax-group/create-tax-group.component.ts

```diff
@@ -0,0 +1,127 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormBuilder, Validators, FormArray } from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
+import { MatDialog } from '@angular/material';
+import { DatePipe } from '@angular/common';
+
+/** Custom Components */
+import { CancelDialogComponent } from 'app/shared/cancel-dialog/cancel-dialog.component';
+
+/** Custom Services */
+import { ProductsService } from 'app/products/products.service';
+
+/**
+ * Create tax-group component.
+ */
+@Component({
+  selector: 'mifosx-create-tax-group',
+  templateUrl: './create-tax-group.component.html',
+  styleUrls: ['./create-tax-group.component.scss']
+})
+
+export class CreateTaxGroupComponent implements OnInit {
+
+  /** Minimum start date allowed. */
+  minDate = new Date(new Date().setFullYear(new Date().getFullYear() - 20));
```

Comment:
min date should be defined as minDate = new Date(2000, 0, 1); because in most of the components it is defined as this only.